### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by altvnv**
Added withdraw when creating multisig wallet, fixed internal transactions.

### CODE REVIEW

**Comment by Enmk** [comment]
Ok

File: contracts/MultiSigWalletFactory.sol [hunk 1]
```diff
@@ -6,6 +6,22 @@ import "./MultiSigWallet.sol";
 /// @title Multisignature wallet factory - Allows creation of multisig wallet.
 /// @author Stefan George - <stefan.george@consensys.net>
 contract MultiSigWalletFactory is Factory {
+    address public _owner;
```
Comment (PashaKlybik) [inline contracts/MultiSigWalletFactory.sol]: Rename `_owner => _creator`

**Review by PashaKlybik — CHANGES_REQUESTED** [review]
Please fix before merging

**Review by PashaKlybik — APPROVED** [review]
Good!

**Pull request merged**